docs/frontends/queues.md

```diff
@@ -189,21 +195,20 @@ It uses a counter `i` and instantiates, in turn, a binary heap that accepts 64-b
 The source code is available in [`stable_binheap.py`][stable_binheap.py].
 
 [builder]: ../builder/calyx-py.md
```

Comment:
Thanks a lot for doing a careful job with the docs! Historically we have a bad habit of changing the file structure but ignoring the docs, leading to a buncha broken links!